addons/base_import/models/base_import.py

```diff
@@ -1459,9 +1464,39 @@ def execute_import(self, fields, columns, options, dryrun=False):
         # convert load's internal nextrow to the imported file's
         if import_result['nextrow']: # don't update if nextrow = 0 (= no nextrow)
             import_result['nextrow'] += skip
+        if additional_filenames:
+            import_result['additional_filenames'] = additional_filenames
 
         return import_result
 
+    def _extract_additional_filenames(self, import_fields, data, model=False, prefix='', additional_filenames=False):
+        model = model or self.res_model
+        additional_filenames = additional_filenames or {}
+        all_fields = self.env[model].fields_get()
+        for name, field in all_fields.items():
```

Comment:
Not need to call `fields_get` which compute and returns a tons of data (`_parse_import_data_recursive` is actually a bad example). Directly use the field objects.
```suggestion
        for name, field in self.env[model]._fields.items():
```

field is now a Field instead of a dictionarry then replace `field['type']` by `field.type`, etc

Reply:
@ryv-odoo 
I have an error on the runbot: [AttributeError: 'One2many' object has no attribute 'relation'. Did you mean: 'relational'?](https://runbot.odoo.com/runbot/build/68279688)

```python
# Old code
field['relation'] # from self.env[model].fields_get()
# New code (make error)
field.relation # from self.env[model]._fields
```

What can I use instead to have the relational model ?
* `field.comodel_name`
* `field._description_relation`
* `field._related_comodel_name`

Reply:
`comodel_name` is equivalent.
see https://github.com/odoo/odoo/blob/0153fff1b3c06bccec352c5a51270dafb77b95d2/odoo/fields.py#L3017